pom.xml

```diff
@@ -175,6 +175,8 @@
             --add-opens=java.base/java.security=ALL-UNNAMED
             --add-opens=java.base/java.util=ALL-UNNAMED
             --add-opens=java.base/java.util.concurrent=ALL-UNNAMED
+            --add-opens=java.base/javax.crypto=ALL-UNNAMED
+            --add-opens=java.base/sun.security.validator=ALL-UNNAMED
```

Comment:
If these are only needed by this one test then we need to organize things so these 2 added add-opens are only applied in that testsuite module, and ideally only for a surefire execution that runs that test. But for now please just limit the addition to testsuite/integration/basic.

Reply:
Done.